feature-configs/demo/sriov/dpdk-network.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: dpdk-network
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |
+    {
+      "type": "dhcp"
+    }
+  networkNamespace: dpdk-testing
```

Comment:
Do we need the creation of the namespace somewhere here?

Reply:
in this file take a look below :)

Reply:
Lol sorry I was looking for another file